app/views/layouts/application.html.erb

```diff
@@ -4,12 +4,22 @@
     <title>MembersOnly</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
-
   <body>
     <%= yield %>
+    <% if flash[:notice] %>
+      <p class="notice"><%= notice %></p>
+    <% end %>
+    <% if flash[:alert]%>
+      <p class="alert"><%= alert %></p>
+    <% end %>
+    <p>
+      <% if user_signed_in? %>
+      <%= link_to "Logout", destroy_user_session_path, method: :delete, class:"button is-info" %>
+   
+    <% end %>
```

Comment:
* [ ] **_There should be NO application logic in the views (no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...)._**
* [ ] **_View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb)_**
* [ ] **_There should be absolutely NO logic in views._**